enviro/compute/fitting.py

```diff
@@ -435,18 +448,25 @@ def _get_distribution(self, dimension, samples, **kwargs):
                                                    samples[1] -> second variable
                                                    ...
 
+
         Returns
         -------
-        distribution : ParametricDistribution instance,
-            the fitted distribution instance
+        distribution : Distribution,
+            The fitted distribution instance.
 
-        dependency : ?
+        dependency : list,
+            List that contains the used dependencies for fitting.
 
-        dist_points: ?
+        dist_points: list,
```

Comment:
List of length three, with one sub-list for each parameter (shape, loc, scale). The sub-lists have a length equal to the number of used intervals. They contain sub-sub-lists. The sub-sub-lists contain the used samples.